README.md

```diff
@@ -41,29 +47,25 @@ The sample app uses the below environmental variables.
 BW_ACCOUNT_ID                        # Your Bandwidth Account Id
 BW_USERNAME                          # Your Bandwidth API Username
 BW_PASSWORD                          # Your Bandwidth API Password
-BW_NUMBER                            # The Bandwidth phone number involved with this application
-USER_NUMBER                          # The user's phone number involved with this application
-BW_VOICE_APPLICATION_ID              # Your Voice Application Id created in the dashboard
-BW_MESSAGING_APPLICATION_ID          # Your Messaging Application Id created in the dashboard
-BASE_CALLBACK_URL                    # Your public base url to receive Bandwidth Webhooks. No trailing '/'
-LOCAL_PORT                           # The port number you wish to run the sample on
+BW_MESSAGING_APPLICATION_ID          # Your Messaging Application Id created in the dashboard]
 ```
 
 # Callback URLs
 
-For a detailed introduction, check out our [Bandwidth Product Specific Callbacks](https://dev.bandwidth.com/docs/messaging/webhooks) page.
+For a detailed introduction, check out our [Bandwidth Messaging Callbacks](https://dev.bandwidth.com/docs/messaging/webhooks) page.
 
 Below are the callback paths:
-* **Should follow `/callbacks/{direction}/{service}` conventions**
-* `<add other callbacks>`
+* `/callbacks/outbound/messaging/status` For Outbound Status Callbacks
+* `/callbacks/inbound/messaging` For Inbound Message Callbacks
 
 ## Ngrok
 
 A simple way to set up a local callback URL for testing is to use the free tool [ngrok](https://ngrok.com/).  
-After you have downloaded and installed `ngrok` run the following command to open a public tunnel to your port (`$LOCAL_PORT`)
+After you have downloaded and installed `ngrok` run the following command to open a public tunnel to your port (in this case port 5001). 
 
-```sh
-ngrok http $LOCAL_PORT
+```cmd
+ngrok http 5001
 ```
+You can view your public URL at `http://127.0.0.1:4040` after ngrok is running.  You can also view the status of the tunnel and requests/responses here.
```

Comment:
`http://127.0.0.1:4040`

Is the port here also supposed to be `5001`?

Reply:
no that's the one generated by ngrok. it's always 4040.

Reply:
ah perfect